dll/win32/msutb/msutb.cpp

```diff
@@ -4729,6 +4712,10 @@ LONG MyWaitForInputIdle(DWORD dwThreadId, DWORD dwMilliseconds)
     if (g_pTipbarWnd && (g_pTipbarWnd->m_dwShowType & TF_SFT_DESKBAND))
         return 0;
 
+// FIXME
+#ifndef STATUS_TIMEOUT
+#define STATUS_TIMEOUT 0x00000102
```

Comment:
This one should be in standard ntstatus.h file, which is itself included from some standard header ?

Reply:
Deleted.